ee/codegen_integration/fixtures/simple_inline_subworkflow_node/display_data/simple_inline_subworkflow_node.json

```diff
@@ -378,7 +378,7 @@
   ],
   "output_variables": [
     {
-      "id": "6ab3665f-881d-488b-9124-a6da40136c68",
+      "id": "b38e08c7-904d-4f49-b8fb-56e1eff254d6",
```

Comment:
We were pointing to the output id of the terminal node within the subworkflow rather than the outer terminal node of the parent workflow. This was an easy mistake since this part of the fixture is hand-crafted.